src/components/PopupDelete

```diff
@@ -0,0 +1,18 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithDelete extends Popup {
+  constructor({ popupSelector, handleBtnClick }) {
+    super({ popupSelector });
+    this._handleBtnClick = handleBtnClick;
+    this._popupModal = document.querySelector(popupSelector);
```

Comment:
**No** need to search the **modal** again in the inheritors of class `Popup` because the parent has already found it in its constructor as `this._popupModal`. It’s available in any child.